packages/models/src/codeObjectId.js

```diff
@@ -1,8 +1,12 @@
 import { CodeObjectType } from './codeObjectType';
 
 export default function codeObjectId(codeObject, tokens = []) {
-  if (codeObject.parent) {
-    codeObjectId(codeObject.parent, tokens);
+  const { parent } = codeObject;
+
+  // If it's a route or query we don't need to include the parent name because it's always the same
+  // ('HTTP server requests' for route and 'Database' for queries). This mirrors the VS Code implementation.
```

Comment:
Can you add a unit test to assert these are as expected?

Reply:
Yup, good idea :+1: